at-tanvo/Exercise1.rb

```diff
@@ -0,0 +1,11 @@
+def non_reapeating
+  a = "asic"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/47f87600d63703e4dcdf4c1477df3685adc982eb/at-tanvo/Exercise1.rb#L2"></span>Style/StringLiterals: Prefer single-quoted strings when you don’t need string interpolation or special symbols. (https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals)</td>
    </tr>
  </tbody>
</table>